pkg/cmd/balances.go

```diff
@@ -19,41 +17,19 @@ func init() {
 
 // go run ./cmd/bbgo balances --session=ftx
 var balancesCmd = &cobra.Command{
-	Use:          "balances",
+	Use:          "balances --session SESSION",
 	Short:        "Show user account balances",
 	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		ctx := context.Background()
 
-		configFile, err := cmd.Flags().GetString("config")
-		if err != nil {
-			return err
-		}
-
-		if len(configFile) == 0 {
-			return errors.New("--config option is required")
-		}
-
 		sessionName, err := cmd.Flags().GetString("session")
 		if err != nil {
 			return err
 		}
 
-		// if config file exists, use the config loaded from the config file.
-		// otherwise, use a empty config object
-		var userConfig *bbgo.Config
-		if _, err := os.Stat(configFile); err == nil {
-			// load successfully
-			userConfig, err = bbgo.Load(configFile, false)
-			if err != nil {
-				return err
-			}
-		} else if os.IsNotExist(err) {
-			// config file doesn't exist
-			userConfig = &bbgo.Config{}
-		} else {
-			// other error
-			return err
+		if userConfig == nil {
```

Comment:
instead of checking here, maybe add the following line in `init()`:
```go
balancesCmd.MarkFlagRequired("config")
```

Reply:
also according to the logic in `root.go`
if config is given, userConfig won't be nil (if file not exists, will only be an empty `bbgo.Config`)

Reply:
good idea